pkg/kp/config.go

```diff
@@ -41,6 +43,6 @@ func NewConsulClient(opts Options) *api.Client {
 	}
 
 	// error is always nil
-	ret, _ := api.NewClient(conf)
-	return ret
+	client, _ := api.NewClient(conf)
```

Comment:
doesn't errcheck only complain if you don't assign it to a variable (even if that variable is _)